BigBlueButton-RLP/Scenes/WebViewController.swift

```diff
@@ -91,15 +97,25 @@ class WebViewController: UIViewController, WKUIDelegate {
         webView.load(URLRequest(url: baseURL))
     }
     
-    private func runJavascript() {
+    private func runJavascript(_ script: String) {
         // Fire event to execute javascript
-        webView.evaluateJavaScript(Constants.fireJSEvent) { (_, error) in
+        webView.evaluateJavaScript(script) { (_, error) in
             if error != nil {
                 print("⚡️☠️ Error executing injected javascript script ☠️⚡️")
             }
         }
     }
     
+    private func getJSessionId() {
```

Comment:
It's unusual to have a function start with `get` and not have it return anything. What would be a better name?

Reply:
Yes, we can name it saveSessionCookie() or storeSessionCookie()?

Reply:
Both fine with me. 👍